pkg/api/norman/server/managementstored/setup.go

```diff
@@ -780,8 +780,9 @@ func KontainerDriver(schemas *types.Schemas, management *config.ScaledContext) {
 		MetadataHandler:       metadataHandler,
 	}
 	lh := kontainerdriver.ListHandler{
-		SysImageLister: management.Management.RkeK8sSystemImages("").Controller().Lister(),
-		SysImages:      management.Management.RkeK8sSystemImages(""),
+		SysImageLister:  management.Management.RkeK8sSystemImages("").Controller().Lister(),
+		SysImages:       management.Management.RkeK8sSystemImages(""),
+		ConfigMapLister: management.Core.ConfigMaps("cattle-system"),
```

Comment:
This is not a lister. Lister is the cache lister which is `management.Core.ConfigMaps("").Controller().Lister()`. Also you might want to create a client in order to CRUD configmaps.

```
ConfigMap: management.Core.ConfigMaps("")
```